src/osd/OSDMap.cc

```diff
@@ -3551,7 +3551,41 @@ void OSDMap::dump_osd(int id, Formatter *f) const
   f->close_section();
 }
 
-void OSDMap::dump(Formatter *f) const
+void OSDMap::dump_pool(CephContext *cct,
+		       int64_t pid,
+                       const pg_pool_t &pdata,
+		       ceph::Formatter *f) const
+{
+  std::string name("<unknown>");
+  const auto &pni = pool_name.find(pid);
+  if (pni != pool_name.end())
+    name = pni->second;
+  f->open_object_section("pool");
+  f->dump_int("pool", pid);
+  f->dump_string("pool_name", name);
+  pdata.dump(f);
+  if (pdata.is_replicated()) {
+    // Add wlb section with values for score, optimal score, raw score
+    //       // and primary_affinity average
+    OSDMap::read_balance_info_t rb_info;
+    auto rc = calc_read_balance_score(cct, pid, &rb_info);
+    if (rc >= 0) {
+      f->open_object_section("read_balance");
+      f->dump_float("score (acting)", rb_info.acting_adj_score);
+      f->dump_float("score (stable)", rb_info.adjusted_score);
+      f->dump_float("optimal_score", rb_info.optimal_score);
+      f->dump_float("raw_score (acting)", rb_info.acting_raw_score);
+      f->dump_float("raw_score (stable)", rb_info.raw_score);
```

Comment:
This will make it easier to references these values as keys in the OSD map.
```suggestion
      f->dump_float("score_acting", rb_info.acting_adj_score);
      f->dump_float("score_stable", rb_info.adjusted_score);
      f->dump_float("optimal_score", rb_info.optimal_score);
      f->dump_float("raw_score _acting", rb_info.acting_raw_score);
      f->dump_float("raw_score_stable", rb_info.raw_score);
```

Reply:
Done